test/support/lib/parser_utils.ex

```diff
@@ -1,39 +1,42 @@
+defmodule User do
+  defstruct is_evil: false, user_id: 0, number_of_hairs_on_head: 0, amount_of_red: 0, nineties_era_color: 0, mint_gum: 0.0, username: "", friends: [], my_map: %{}, blocked_user_ids: MapSet.new(), optional_integers: []
+end
+
+defmodule Nesting do
+  defstruct user: nil, nested: nil
+end
+
+defmodule Shared.SharedStruct do
+  defstruct key: nil, value: nil
+end
```

Comment:
These are tightly coupled to specific test fixtures. It would be nice if we could use the code generator to get them dynamically.